src/experimental/tools/videoThumbnailLoader/init_source_buffer.ts

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright 2015 CANAL+ Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  EMPTY,
+  Observable,
+  of as observableOf,
+} from "rxjs";
+import {
+  catchError,
+  filter,
+  map,
+  mapTo,
+  mergeMap,
+  shareReplay,
+} from "rxjs/operators";
+import createSegmentLoader from "../../../core/fetchers/segment/create_segment_loader";
+import { QueuedSourceBuffer } from "../../../core/source_buffers";
+import dash from "../../../transports/dash";
+import prepareSourceBuffer from "./prepare_source_buffer";
+import { IContentInfos } from "./types";
+
+const _currentVideoSourceBuffers: WeakMap<HTMLMediaElement,
+                                          QueuedSourceBuffer<Uint8Array>> = new WeakMap();
+const _currentContentInfos: WeakMap<HTMLMediaElement,
+                                    IContentInfos> = new WeakMap();
+
+const { loader, parser } = dash({ lowLatencyMode: false }).video;
```

Comment:
We shouldn't be relying on `dash` directly here. Isn't the same segment loader/parser than the one for the Manifest (which already has been imported) sufficient?

Reply:
?